static/Home Assistant/creating-zones.md

```diff
@@ -32,6 +32,30 @@ Select the add-on, then hit install. Once installed, hit start and then "Open We
 
 ![Everything Presence Zone Configurator Dashboard](../images/home-assistant-creating-zones-zone-configurator-dashboard.png)
 
+### (Advanced) Alternative install with docker (no Supervisor)
+
+1. clone the git repository `git clone ttps://github.com/EverythingSmartHome/everything-presence-addons`
```

Comment:
There isn't a need to do step 1-3 as this is published to Docker hub. So you can just build it directly using the following instead without cloning:

`docker run -d -e HA_URL=http://HA_URL:8123 -e HA_TOKEN=LONG_LIVE_TOKEN -p 8099:8099 --name everything-presence-mmwave-configurator everything-presence-mmwave-configurator:latest `